src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -163,7 +163,7 @@ private function parseDefinitions($content, $file)
      */
     private function parseDefinition($id, $service, $file)
     {
-        if (is_string($service) && 0 === strpos($service, '@')) {
+        if (is_string($service) && $service[0] && '@' === $service[0]) {
```

Comment:
What should be used instead ?
- ``'' !== $service``
- ``!empty($service)``
- ``strlen($service)``
- ``isset($service[0])``